### ISSUE TITLE
merge reps into devtools-core/packages

### ISSUE THREAD

**Issue by gasolin** [devtools-html/reps#124]
I'm writing docs for developing the Network Monitor panel, and I'd like to add a section about `npm link` required modules to work on cutting-edge features. Here are modules required for the Network Monitor and is hosted under `devtools-html`

* devtools-config
* devtools-launchpad
* devtools-modules
* devtools-source-editor
* devtools-splitter
* **devtools-reps**

For now I have to tell the developer: clone both `devtools-html/devtools-core` and `devtools-html/reps` then do `npm link` for each package in different path. 

From the documentation view I can not tell the developer why they have to clone a separate `reps` repo for `devtools-reps`. 

Once reps merged into devtools-core/packages, the way to publish `devtools-reps` npm package is not changed, the procedure to merge to firefox can be the same, but we will have a sane repository to host all of our components.  (and this way save words of explanation)

**Comment by gasolin** [comment]
@juliandescottes @tromey @jasonLaster how do you think?

**Comment by nchevobbe** [comment]
Blocked by https://github.com/devtools-html/reps/pull/157 & https://github.com/devtools-html/reps/pull/158

**Comment by nchevobbe** [comment]
Done in https://github.com/devtools-html/devtools-core/pull/407
